### ISSUE TITLE
Remove "The Ember Times" from the title of each blog.

### ISSUE THREAD

**Issue by mansona** [ember-learn/ember-blog#904]
If you visit a single post of the ember times you should see the Ember Times logo and then just the title of the blog `Issue No. 166`

Example: https://blog.emberjs.com/the-ember-times-issue-166

This should still have `the-ember-times-issue-166` in the url and the meta title should also have "The Ember Times" in it :+1:

**Comment by ijlee2** [comment]
I don't think this is a good idea. The logo image has `alt=""` and `role="presentation"` to indicate that it's decorative. We'd also need to update every post on `dev.to`.

**Comment by mansona** [comment]
This was something that came out of a last-minute review of the Blog before the new design went live. I think it was @pichfl 's idea so maybe he can explain more the rationale why we should consider it. If I remember correctly it was something to do with too much repetition 🤔 

I'm not married to this idea so if we agree so if we discuss it and agree that it's not ok or we don't want to do it then that's ok too 👍
